packages/devtools_app/test/cpu_profile_model_test.dart

```diff
@@ -3,9 +3,13 @@
 // found in the LICENSE file.
 
 // ignore_for_file: avoid_redundant_argument_values
+// ignore_for_file: import_of_legacy_library_into_null_safe
```

Comment:
What's not null safe here? I thought everything except the Provider page has been migrated already?

Reply:
looks like devtools_test is the culprit